storybook/src/docs/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -232,4 +234,109 @@ storiesOf("stories/components/DataGrid", module)
                 <DataGrid rows={exampleRows} columns={columns} />
             </Box>
         );
+    })
+    .add("useDataGridExcelExport", () => {
+        const dataGridProps = useDataGridRemote();
+
+        const variables = {
+            limit: dataGridProps.pageSize,
+            offset: dataGridProps.page * dataGridProps.pageSize,
+            sort: dataGridProps.sortModel[0]?.field,
+            order: dataGridProps.sortModel[0]?.sort,
+        };
+
+        const columns: GridColDef[] = [
+            {
+                field: "mission_name",
+                headerName: "Mission Name",
+            },
+            {
+                field: "launch_date_local",
+                headerName: "Launch Date",
+                type: "dateTime",
+            },
+        ];
+
+        type Launch = {
+            id: string;
+            mission_name: string;
+            launch_date_local: Date;
+        };
+
+        interface LaunchesPastResultData {
+            data: Launch[];
+            result: { totalCount: number };
+        }
+
+        interface QueryVariables {
+            offset: number;
+            limit: number;
+            sort?: string;
+            order?: string | null;
+        }
+
+        interface GQLQuery {
+            __typename?: "Query";
+            launchesPastResult: LaunchesPastResultData;
+        }
```

Comment:
I'm not a huge fan of all the types here in the story because they distract from the actual story. Can we maybe move them outside this particular story?